lib/active_merchant/billing/gateways/card_stream.rb

```diff
@@ -184,7 +184,16 @@ def refund(money, authorization, options = {})
         add_pair(post, :xref, authorization)
         add_amount(post, money, options)
         add_remote_address(post, options)
-        commit('REFUND', post)
+        response = commit('REFUND_SALE', post)
+
+        return response if response.success?
+        return response unless options[:force_full_refund_if_unsettled]
```

Comment:
What's the case for relying on this flag instead of voiding by default if unsettled?

Reply:
My first pass at this didn't include the flag. I only added it in when I saw that other adapters (those mentioned in the commit message) use `force_full_refund_if_unsettled` to determine if void should be called. I tend to favor existing conventions for consistency sake, but I can see a case for not including the flag.

Reply:
Interesting, I hadn't seen those in use before... It doesn't look like Spreedly utilizes them? 

I also recall another update recently that fixed a very similar issue where a transaction may be unsettled and need to be voided instead of refunded, but I can't seem to find it... I don't think that used a flag like this.

Reply:
No, we don't utilize the the force flag.

Hmmm maybe you're thinking of https://github.com/activemerchant/active_merchant/pull/2772?

Reply:
Ah, yes. So perhaps that should use this flag if we want to begin using it...

Reply:
@curiousepic - Commented on the `force_full_refund_if_unsettled` flag on #2772. Any other concerns with this one?